.circleci/config.yml

```diff
@@ -118,6 +118,8 @@ jobs:
       - run:
           name: Run end-to-end tests
           command: |
+            mkdir -p "binaryData/Connectomics department"
+            chmod 777 "binaryData/Connectomics department"
```

Comment:
below it looks like the new test actually uses Organization_X. Why is it necessary to create the Connectomics department directory here?

Reply:
Hm, I think that is a legacy artifact, from when things were named differently. Could you name it Organization_X there too and see if that breaks anything?